datahub-web-react/src/app/entity/group/GroupOwnerSideBarSection.tsx

```diff
@@ -50,16 +50,17 @@ export default function GroupOwnerSideBarSection({ urn, ownership, refetch }: Pr
                     </AddOwnerButton>
                 )}
             </SectionWrapper>
-            <AddOwnersModal
-                urn={urn}
-                hideOwnerType
-                type={EntityType.CorpGroup}
-                visible={showAddModal}
-                refetch={refetch}
-                onCloseModal={() => {
-                    setShowAddModal(false);
-                }}
-            />
+            {showAddModal && (
```

Comment:
Why not use the visible flag?

Reply:
responded to this and the few below on the last instance of this type of comment